nums/core/systems/systems.py

```diff
@@ -59,6 +63,16 @@ def register(self, name: str, func: callable, remote_params: dict = None):
     def call(self, name: str, args, kwargs, device_id: DeviceID, options: Dict):
         return self._remote_functions[name](*args, **kwargs)
 
+    def register_actor(self, name: str, cls: type):
+        assert name not in self._actors
```

Comment:
shall we add a error message that the name is not unique?

Reply:
This is low-level, we don't expect users to run this. If we make a mistake we want to know about it ;)